### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by helloWorldCindy**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG
Upgrade Ember-cli to 2.12.3

### CODE REVIEW

**Comment by notmessenger** [comment]
`http` to `https` in _.gitignore_ file

File: .jshintrc [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (notmessenger) [inline .jshintrc]: Remove this file

File: package.json [hunk 2]
```diff
@@ -47,28 +46,26 @@
     "ember-cli-sri": "^2.0.0",
     "ember-cli-test-loader": "^1.1.0",
```
Comment (notmessenger) [inline package.json]: Remove this

File: package.json [hunk 3]
```diff
@@ -79,7 +76,7 @@
     "frost"
   ],
   "dependencies": {
-    "ember-cli-babel": "^5.1.10",
+    "ember-cli-babel": "^5.1.7",
     "ember-cli-htmlbars": "^1.0.1",
```
Comment (notmessenger) [inline package.json]: Wrong version

File: tests/.jshintrc [hunk 4]
```diff
@@ -0,0 +1,52 @@
+{
```
Comment (notmessenger) [inline tests/.jshintrc]: Remove this file

File: tests/dummy/config/environment.js [hunk 5]
```diff
@@ -1,14 +1,18 @@
+/* eslint-env node */
 module.exports = function (environment) {
```
Comment (notmessenger) [inline tests/dummy/config/environment.js]: Remove `ENV.rootURL = '/'` in test configuration

Comment (helloWorldCindy) [inline tests/dummy/config/environment.js]: Fixed.

File: tests/.jshintrc (hunk 4 above)
Comment (helloWorldCindy) [inline tests/.jshintrc]: Fixed

File: package.json (hunk 3 above)
Comment (helloWorldCindy) [inline package.json]: Fixed.

File: package.json (hunk 2 above)
Comment (helloWorldCindy) [inline package.json]: Fixed.

File: .jshintrc (hunk 1 above)
Comment (helloWorldCindy) [inline .jshintrc]: Fixed.

**Comment by notmessenger** [comment]
👍

**Pull request merged**